src/views/Home/components/ChapelProgress/index.js

```diff
@@ -45,16 +45,16 @@ export default class ChapelProgress extends Component {
     } else {
       content = <Doughnut data={data} />;
       if (current === 1) {
-        subheader = `${current} CL&W`;
+        subheader = `${current} CL&W credits`;
```

Comment:
This should be singular ("credit" not "credits") because it is in the clause `current === 1`, meaning that there is only one credit.